draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -611,6 +616,9 @@ Note that in this case, the initial and trailing DQUOTE characters are not
 stripped. They are part of the cookie-value, and will be included in Cookie
 header fields sent to the server.
 
+Per the grammar above, cookie-avs MUST NOT contain leading or trailing WSP
+characters as they will be interpreted as BWS and removed.
+
```

Comment:
This is to address the first issue in the review regarding whitespace around cookie attributes.